src/seth/libexec/seth/seth-source

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+### seth-source -- fetch the contract source from etherscan
+### Usage: seth source <address> [<options>]
+###
+### Requires an etherscan api key
+###
+set -e
+[[ $ETHERSCAN_API_KEY ]] || {
+  cat >&2 <<.
+
+  You need an Etherscan Api Key to verify contracts.
+  Create one at https://etherscan.io/myapikey
+
+  Then export it with \`export ETHERSCAN_API_KEY=xxxxxxxx'
+
+.
+  exit 1
+}
+
+ADDR=$(seth --to-address "$1")
+
+RESPONSE=$(seth curl "http://api-cn.etherscan.com/api?module=contract&action=getsourcecode&address=""$ADDR""&apikey=""$ETHERSCAN_API_KEY""" | seth --show-json)
```

Comment:
I think this works too?

```suggestion
RESPONSE=$(seth curl "http://api-cn.etherscan.com/api?module=contract&action=getsourcecode&address=${ADDR}&apikey=${ETHERSCAN_API_KEY} | seth --show-json)
```

Reply:
shellcheck likes this? shellcheck made me double quote